src/sentry/conf/server.py

```diff
@@ -1329,11 +1329,12 @@ def create_partitioned_queues(name):
 
 # Rate limiting backend
 SENTRY_RATELIMITER = "sentry.ratelimits.base.RateLimiter"
+SENTRY_RATELIMITER_ENABLED = False
 SENTRY_RATELIMITER_OPTIONS = {}
 # These values were determined from analysis on one week of api access logs
 SENTRY_RATELIMITER_DEFAULT_IP = 1000
 SENTRY_RATELIMITER_DEFAULT_USER = 3230
-SENTRY_RATELIMITER_DEFAULT_ORG = 620
+SENTRY_RATELIMITER_DEFAULT_ORG = 3230
```

Comment:
can we make user `620`? Also, is this overridable without deploy?

Reply:
Dropped them down. The settings aren't overridable without deploy at the moment. We probably need a new system to allow that.